libarchive/__init__.py

```diff
@@ -28,85 +28,124 @@
 import sys
 import time
 import warnings
-
 from libarchive import _libarchive
 from io import StringIO
 
-PY3 = sys.version_info[0] == 3
+PY3 = sys.version_info[0] >= 3
 
 # Suggested block size for libarchive. Libarchive may adjust it.
 BLOCK_SIZE = 10240
 
-MTIME_FORMAT = ''
+MTIME_FORMAT = ""
```

Comment:
It seems like this whole file was just run through some kind of auto formatter. Can you limit the changes to only what is relevant to this change?

Reply:
Sorry about that...
I'm newbie with VSCode...  It proposed to install some python linting plugins and I've accepted.
I don't really know how to go back and undo formatting it did....

Reply:
I've fixed the formatinng issue